parsing/parser.mly

```diff
@@ -1483,7 +1482,7 @@ class_signature:
   | LET OPEN override_flag attributes mkrhs(mod_longident) IN class_signature
       { mkcty ~loc:$sloc ~attrs:$4 (Pcty_open($3, $5, $7)) }
 ;
-class_sig_body:
+%inline class_sig_body:
```

Comment:
I believe that `class_self_type -> class_sig_body` form an instance of the ordering-conflict pattern `lin+br -> nonlin+nonbr`, and that my previous reasoning would suggest inlining `class_self_type` instead.